motoko/vetkd/README.md

```diff
@@ -21,38 +15,33 @@ Additionally, the repository provides:
 
 The implementation of [the proposed vetKD system API](https://github.com/dfinity/interface-spec/pull/158) used in this example is **unsafe**, e.g., we hard-code a master secret key, rather than using a master secret key that is distributed among sufficiently many Internet Computer nodes through distributed key generation. **Do not use this in production or for sensitive data**! This example is solely provided **for demonstration purposes** to collect feedback on the mentioned vetKD system API. See also the respective disclaimer [in the system API canister implementation](https://github.com/dfinity/examples/blob/master/rust/vetkd/src/system_api/src/lib.rs#L19-L26).
 
-## Prerequisites
-- [x] Install the [IC SDK](https://internetcomputer.org/docs/current/developer-docs/setup/install/).
-- [x] Install [Node.js](https://nodejs.org/en/download/).
+### Prerequisites
```

Comment:
This Prerequisites section is now a sub-section of the Disclaimer section, which seems odd.

Reply:
Added a separator character to clearly distinguish the Disclaimer section from the other sections.